auth-web/src/assets/scss/base.scss

```diff
@@ -66,11 +66,11 @@ h1 {
 }
 
 h2 {
-  font-size: 1.75rem;
+  font-size: 1.5rem;
```

Comment:
Editing the base styles will have the impact on the overall application, I hope this is necessary?